### DISCUSSION TITLE
Fooocus crashes with no error message.  Possibly Pytorch Version issue?

### DISCUSSION THREAD

**Discussion by Tanoshimi** [lllyasviel/Fooocus#3525] [General]
So, I had Fooocus_win64_2-5-0 working for a long time.  I added Tortoise-TTS to my system, and also Stable Diffusion.  Apparently, each of these has different requirements for what version of torch it uses, and now Fooocus just exits without giving me any kind of message.  I'd be happy to roll back to whatever version of Pytorch it needs, but I don't know what that it.  Also, is there a log file somewhere I can find what's crashing it?  This is what I get:

Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE

D:\AI Apps\Fooocus_win64_2-5-0>pause
Press any key to continue . . .

**Comment by Tomcreating** [comment]
Did you find a fix for this? I am getting the same error. Deleted and reinstalled. Same thing. I was just running it this week.

**Reply by Tanoshimi** [reply]
No.  After not figuring it out for a long, long time, I just stopped using it.  Unfortunate, because I was enjoying it.  But as I said in my OP, I think it's related to a wrong version of pyTorch.  Might want try upgrading/downgrading that.  Or looking for a previous version of Fooocus to install.
